uaclient/config.py

```diff
@@ -681,6 +680,51 @@ def help(self, name):
         response_dict["help"] = help_ent.help_info
         return response_dict
 
+    def process_config(self):
```

Comment:
Just to be sure, we are treating `apt` differently from `snap`  and `livepatch`  because by default, the proxy configurations will be empty and we don't want to disable any existing `snap`and `livepatch` proxies in that scenario, right ?

Reply:
Right. We treat snap/livepatch differently because we don't want to accidentally remove a manually set proxy for those services. We couldn't remove a manually set apt proxy even if we tried, so we don't have to message about it and can just worry about our own apt.conf.d file.